### ISSUE TITLE
Can you provide a tf.example as input example?

### ISSUE THREAD

**Issue by hudengjunai** [serizba/cppflow#193]
my model is like this

> MetaGraphDef with tag-set: 'serve' contains the following SignatureDefs:

signature_def['serving_default']:
  The given SavedModel SignatureDef contains the following input(s):
    inputs['examples'] tensor_info:
        dtype: DT_STRING
        shape: (-1)
        name: input_example_tensor:0
  The given SavedModel SignatureDef contains the following output(s):
    outputs['prob'] tensor_info:
        dtype: DT_FLOAT
        shape: (-1)
        name: Sigmoid:0
  Method name is: tensorflow/serving/predict

**Comment by ns-wxin** [comment]
Did you get any response? Your question is similar to mine which is about creating DT_STRING tensor. Also my address sanitizer is complaining memory leak on the way the DT_STRING tensor is created on the function "TF_TString_Copy()".

**Comment by hudengjunai** [comment]
No, I'am trying use compiled tensorflow_cc.so directly from tensorflow-source. and use the tf c++ header and lib. 
```
RUN cd tensorflow-1.15.0 && ./configure && \
        bazel build  //tensorflow/tools/pip_package:build_pip_package && \
        bazel build //tensorflow:libtensorflow_cc.so && \
        bazel build --config=opt tensorflow:install_headers

```

After compiled the libtesorflow_cc.so lib and header files. 
Encountered some protobuf version problem. with grpc-server. So I delay this feature for my project.

> Did you get any response? Your question is similar to mine which is about creating DT_STRING tensor. Also my address sanitizer is complaining memory leak on the way the DT_STRING tensor is created on the function "TF_TString_Copy()".

**Comment by serizba** [comment]
Hi,

So, the problem is about how to create a proper TF_TString tensor, right?
